data/json/items/containers.json

```diff
@@ -524,13 +524,13 @@
         "color" : "light_cyan",
         "name" : "little metal tank",
         "category": "other",
-        "description" : "A little gas tank.  Useful for crafting.",
+        "description" : "A little metal tank for holding gas or liquids.  Useful for crafting.",
         "price" : 1000,
-        "weight" : 400,
-        "volume" : 5,
-        "bashing" : 2,
+        "weight" : 800,
+        "volume" : 8,
```

Comment:
The volume of container is also its capacity. In case of extinguishers and oxygen tank, it should be lower, because there are extra parts there.
If you still want it to get the boost, there are few vehicle parts that use little metal tanks. Those should get a proportional boost to capacity too.